README.rst

```diff
@@ -78,35 +78,35 @@ To see what coala can do for your language, run:
 +----------------------------+----------------------------+----------------------------+
-| CMake                      | Matlab/Octave              | Verilog                    |
+| CMake                      | Matlab/Octave              | TypeScript                 |
 +----------------------------+----------------------------+----------------------------+
-| CoffeeScript               | Natural Language (English) | VHDL                       |
+| CoffeeScript               | Natural Language (English) | Verilog                    |
 +----------------------------+----------------------------+----------------------------+
-| CSS                        | Perl                       | Vimscript                  |
+| CSS                        | Perl                       | VHDL                       |
 +----------------------------+----------------------------+----------------------------+
-| Dart                       | PHP                        | XML                        |
+| Dart                       | PHP                        | Vimscript                  |
 +----------------------------+----------------------------+----------------------------+
-| Fortran                    | Python 2                   | YAML                       |
+| Fortran                    | Python 2                   | XML                        |
 +----------------------------+----------------------------+----------------------------+
-| Go                         | Python 3                   |                            |
+| Go                         | Python 3                   | YAML                       |
 +----------------------------+----------------------------+----------------------------+
 | Haskell                    | R                          |                            |
 +----------------------------+----------------------------+----------------------------+
 | HTML                       | reStructured Text          |                            |
 +----------------------------+----------------------------+----------------------------+
 | Java                       | Ruby                       |                            |
 +----------------------------+----------------------------+----------------------------+
-| JavaScript                 | Scala                      |                            |
+| JavaScript                 | Rust                       |                            |
 +----------------------------+----------------------------+----------------------------+
-| JSP                        | SCSS                       |                            |
+| JSP                        | Scala                      |                            |
 +----------------------------+----------------------------+----------------------------+
-| Julia                      | sh & bash scripts          |                            |
+| Julia                      | SCSS                       |                            |
```

Comment:
Are we even updating the number of bears for new bears? Most new bears change this, the one I have seen merged atleast